cmd/run/run.go

```diff
@@ -274,6 +275,7 @@ func DefaultConfig() *Config {
 				Endpoint: "0.0.0.0:4317",
 			},
 			SampleRatio: 0.2,
+			ServiceName: "openfga",
```

Comment:
Please add a test for this default in  `TestDefaultConfig` (`cmd/run/run_test.go`).

Reply:
test added